libraries/src/MVC/Model/BaseDatabaseModel.php

```diff
@@ -166,7 +167,8 @@ protected function _getListCount($query)
 			&& $query->group === null
 			&& $query->merge === null
 			&& $query->querySet === null
-			&& $query->having === null)
+			&& $query->having === null
+		)
```

Comment:
Unrelated to PR content - best to leave this alone in this file for now.

```suggestion
			&& $query->having === null)
```